.github/workflows/code.yml

```diff
@@ -11,10 +11,10 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: true
-      matrix:
+      matrix: # https://www.postgresql.org/docs/release
         # always use latest Python b/c the point here is to test PostgreSQL compatibility
-        python-version: ["3.11"]
-        postgres-version: [12, 13, 14, 15, 16]
+        python-version: [3.x]
```

Comment:
Please set this to a version number not just 3.x

Reply:
```suggestion
        python-version: ["3.12"]
```